libraries/audio/src/AudioHRTF.cpp

```diff
@@ -91,8 +91,8 @@ static const float azimuthTable[NAZIMUTH][3] = {
 // A first-order shelving filter is used to minimize the disturbance in ITD.
 //
 // Loosely based on data from S. Spagnol, "Distance rendering and perception of nearby virtual sound sources
-// with a near-field filter model,� Applied Acoustics (2017)
```

Comment:
This here is unrelated to the PR, but VSCode wasn't saving it correctly, so I had to get rid of it.